core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -721,7 +721,9 @@ class HttpGoogleServicesDAO(
         blocking {
           val span = startSpanWithParent("executeGoogleRequest", s)
           span.putAttribute("googleProjectId", AttributeValue.stringAttributeValue(googleProjectId.value))
-          span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName))
+          if (projectBillingInfo.getBillingAccountName != null) {
+            span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName))
```

Comment:
Nit: A cleaner thing to do here might be:
```
span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName match {
            case value => value
            case null => ""
          }))```

As opposed to doing the null check.

Reply:
You may need to swap the cases or change that slightly, I don't know for sure.

Reply:
I think we need to wrap in an option or use a stable name. I'll wrap in an option